scripts/compile.ts

```diff
@@ -34,34 +31,36 @@ if (import.meta.main) {
       "../src/daemon.ts",
     ],
     cwd: __dirname,
-  });
-  const status = await p.status();
-  if (!status.success) {
+  })).output();
+  if (!p.success) {
     console.error(
       "Failed to run deno compile for target",
       target,
       "code:",
-      status.code,
+      p.code,
     );
-    Deno.exit(status.code);
+    Deno.exit(p.code);
   }
 
   const binPath = `${__dirname}/../gui/binaries/s3si-${target}${
     Deno.build.os === "windows" ? ".exe" : ""
   }`;
   console.log("Test the binary");
-  const s3si = Deno.run({
-    cmd: [binPath],
+  const s3si = new Deno.Command(binPath, {
     stdin: "piped",
     stdout: "piped",
-  });
-  await s3si.stdin?.write(
+  }).spawn();
+  const s3siWriter = s3si.stdin.getWriter();
+  await s3siWriter.write(
     new TextEncoder().encode(
       '{"jsonrpc":"2.0","method":"hello","params":[],"id":1}\n',
     ),
   );
-  s3si.stdin?.close();
-  const output = new TextDecoder().decode(await s3si.output());
+
+  const output = new TextDecoder().decode(
+    (await s3si.stdout.getReader().read()).value,
+  );
+  await s3siWriter.close();
```

Comment:
https://deno.land/api@v1.29.2?unstable=&s=Deno.Command

>If stdin is set to "piped", the stdin [WritableStream](https://deno.land/api?s=WritableStream) needs to be closed manually.

Reply:
LGTM. Although the Reader of ReadableStream does not guarantee that a read can get everything, but this is just a simple test, let's pass it!